app/src/commonMain/kotlin/it/vercruysse/lemmyapi/v0x19/datatypes/CommunityAggregates.kt

```diff
@@ -13,4 +13,5 @@ data class CommunityAggregates(
     val users_active_week: Int,
     val users_active_month: Int,
     val users_active_half_year: Int,
+    val subscribers_local: Int?,
```

Comment:
subscribers_local and local_only haven't been merged yet, so this throws errors in jerboa like:

`subscribers_local is required but it was missing at path:...`

Reply:
I see, I should ve used the appropriate lemmy-js-client version. But this also shows a fault in my type generation. It is marked nullable(because i want to be optional) but it still required due to not being optional for that, I still need to set default to null.

I'll change that too thx.

Do you happen to know which version is the latest release for lemmy-js-client?

Reply:
> subscribers_local and local_only haven't been merged yet, so this throws errors in jerboa like:

This shouldn't be a problem as they are optional

There  was no fault in my type generation, I did `subscribers_local` manually since it was not optionally and to be backwards compatible it should have been. I forgot to add a default to make it truly optional. I have added a test for this. So it shouldn't happen again.

This should work fine now in `0.2.5`

Reply:
Thx!

`0.19.2-alpha.1` is the current version being used by production lemmy-ui.